migrations/versions/850_user_application.py

```diff
@@ -0,0 +1,34 @@
+"""user application
+
+Revision ID: b48fd3ae3d00
+Revises: 840
+Create Date: 2016-12-15 14:17:27.554023
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '850'
+down_revision = '840'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(u'application_user_id_fkey', 'application', type_='foreignkey')
+    op.drop_column('application', 'user_id')
```

Comment:
Need to copy the existing values to the new column before dropping right?

Something like: 

update "user" u set application_id = a.id
from application a where a.user_id = u.id;